### PR TITLE
Belated Pull Request

### PR DESCRIPTION
**Pull request by dunstontc**
# Belated Pull Request

## Changes:
  - Added initial tests
  - Implemented basic functionality

### CODE REVIEW

File: LoggingKata.Test/ParserTest.cs [hunk 1]
```diff
@@ -10,10 +10,29 @@ namespace LoggingKata.Test
     [TestFixture]
     public class TacoParserTestFixture
     {
+        [Test]
+        public void ShouldReturnNullForEmptyString()
```
Comment (codytwinton) [inline LoggingKata.Test/ParserTest.cs]: Missing tests for null string, no valid number values, and valid number values with no description.

File: LoggingKata/ITrackable.cs [hunk 2]
```diff
@@ -4,5 +4,7 @@ public interface ITrackable
     {
         string Name { get; set; }
         Point Location { get; set; }
+//        double Longitude { get; set; }
```
Comment (codytwinton) [inline LoggingKata/ITrackable.cs]: Delete commented code

File: LoggingKata/Program.cs [hunk 3]
```diff
@@ -1,36 +1,93 @@
...
+using Geolocation;
 using log4net;
-using System.IO;
 
 namespace LoggingKata
 {
     class Program
     {
-        //Why do you think we use ILog?
+        // QUESTION: Why do you think we use ILog?
```
Comment (codytwinton) [inline LoggingKata/Program.cs]: Delete question comments

File: LoggingKata/Program.cs [hunk 4]
```diff
@@ -1,36 +1,93 @@
...
         private static readonly ILog Logger =
             LogManager.GetLogger(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
 
+        // Grab the path from Environment.CurrentDirectory + the name of your file
```
Comment (codytwinton) [inline LoggingKata/Program.cs]: Delete all unnecessary comments

File: LoggingKata/Program.cs [hunk 5]
```diff
@@ -1,36 +1,93 @@
...
             {
                 Console.WriteLine("You must provide a filename as an argument");
                 Logger.Fatal("Cannot import without filename specified as an argument");
                 return;
             }
 
-            Logger.Info("Log initialized");
             var lines = File.ReadAllLines(args[0]);
+            
+            switch (lines.Length)
```
Comment (codytwinton) [inline LoggingKata/Program.cs]: Nice use of the switch statement

File: LoggingKata.Test/ParserTest.cs [hunk 6]
```diff
@@ -5,15 +5,47 @@
...
+
+        [Test]
+        public void ShouldReturnNullForInvalidNumbers()
+        {
+            const string testString = "-84.460905, \"Jaspe... \"";
+            var res = TacoParser.Parse(testString);
+            Assert.Null(res);
+        }
+
+        // TODO: Test for valid number values with no description.
```
Comment (codytwinton) [inline LoggingKata.Test/ParserTest.cs]: You neglected this test
